pkg/cmd/run.go

```diff
@@ -683,6 +704,8 @@ func init() {
 	runCmd.Flags().String("mount-format", "json", fmt.Sprintf("file format to use. if not specified, will be auto-detected from mount name. one of %v", models.SecretsMountFormats))
 	runCmd.Flags().String("mount-template", "", "template file to use. secrets will be rendered into this template before mount. see 'doppler secrets substitute' for more info.")
 	runCmd.Flags().Int("mount-max-reads", 0, "maximum number of times the mounted secrets file can be read (0 for unlimited)")
+	runCmd.Flags().StringSliceVarP(&secretsToInclude, "only-secrets", "i", []string{}, "only includes the specified secrets")
```

Comment:
Since the flag rename, I don't think `-i` is an appropriate short flag anymore. What about `-o`? Or we could just have the long flag name to start.

Reply:
Good callout! Let's remove the shorthand for now, `-o` reads like an output flag to me. I prefer to err on the side of caution and not confuse users.